app/models/user.rb

```diff
@@ -1,4 +1,8 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
```

Comment:
- [ ] Good job so far 👏 👏  In addition to what you have done, I would suggest that you add the `:confirmable`  attribute which will allow for email confirmation when user is trying to sign up as this is among the project requirements 😃 😉 
![image](https://user-images.githubusercontent.com/90222110/217921391-1151f596-0574-41b6-afa1-fc01df9c3f5a.png)

Reply:
I did the confirmation, Thank you